lib/timber/integrations/phoenix_instrumenter.ex

```diff
@@ -138,19 +229,32 @@ defmodule Timber.Integrations.PhoenixInstrumenter do
     :ok
   end
 
+  #
+  # Utility
+  #
+
   @spec get_log_level(atom) :: atom
   defp get_log_level(default) do
     Timber.Config.phoenix_instrumentation_level(default)
   end
 
-  defp params(%{__struct__: :"Elixir.Plug.Conn.Unfetched"}), do: %{}
+  defp filter_params(%{__struct__: :"Elixir.Plug.Conn.Unfetched"}),
+    do: %{}
 
-  defp params(params) when is_list(params) or is_map(params) do
+  defp filter_params(params) when is_list(params) or is_map(params) do
     params
     |> Phoenix.Logger.filter_values()
     |> Enum.into(%{})
   end
 
   # Unknown type, convert to a blank map for now
-  defp params(_params), do: %{}
+  defp filter_params(_params),
+    do: %{}
+
+  defp try_atom_inspect(nil),
+    do: nil
+
+  defp try_atom_inspect(atom) do
+    inspect(atom)
```

Comment:
Please use `Atom.to_string/1` here. `inspect/1,2` as we have seen in our server-side usage is not very particular. I would prefer using something that is guaranteed to just turn an Atom into a string.

As an aside, `Atom.to_string/1` is inlined by the compiler to `:erlang.atom_to_binary/2` which is a built-in function of the VM. That's a good thing for this.